suite-native/module-device-settings/src/screens/ContinueOnTrezorScreen.tsx

```diff
@@ -1,9 +1,14 @@
 import { ContinueOnTrezorScreenContent } from '@suite-native/device';
+import { useHandleHardwareBackNavigation } from '@suite-native/navigation';
 
 import { DeviceInteractionScreenWrapper } from '../components/DeviceInteractionScreenWrapper';
 
-export const ContinueOnTrezorScreen = () => (
-    <DeviceInteractionScreenWrapper>
-        <ContinueOnTrezorScreenContent />
-    </DeviceInteractionScreenWrapper>
-);
+export const ContinueOnTrezorScreen = () => {
+    useHandleHardwareBackNavigation();
```

Comment:
On this screen, it doesn't seem to work well. 

If you go to change pin and swipe back on Continue on Your trezor, there's a bug

Reply:
fixed, sorry I had to rebase so it got lost in the initial commit but the fixed has been applied directly in the iOS hook